packages/app-builder/package.json

```diff
@@ -35,13 +35,13 @@
     "postcss": "^8.4.38",
     "tailwindcss": "3.4.4",
     "tsx": "^4.15.7",
-    "vite": "^5.3.1"
+    "vite": "^5.3.4"
   },
   "dependencies": {
     "@ariakit/react": "^0.4.7",
-    "@conform-to/dom": "^0.9.1",
-    "@conform-to/react": "^0.9.1",
-    "@conform-to/zod": "^0.9.1",
+    "@conform-to/dom": "^1.1.5",
```

Comment:
ditto here

Reply:
I think the comment is not at the good position